docs/ams-schema-spec.bs

```diff
@@ -182,52 +1216,173 @@ Indien het gewenst is om een inverse relatie op te nemen de weergave van de reso
+Voor het definieren van een eenheid wordt standaard een string met de eenheid volgens de [[!UCUM inline]] codering gebruikt.
+
+<pre class=example highlight=json>
+    "unit": "mm/h"
+</pre>
 
-`"unit": "mm/h"`
+Note: Zie [deze link](https://ucum.org/trac/wiki/adoption/common) voor een overzicht van veelgebruikte eenheden in UCUM formaat.
 
-UCUM heeft geen expliciete ondersteuning voor valuta dus gebruiken we unit annotaties ([ISO 4217 code](https://unitsofmeasure.org/ucum.html|UCUM 2.1§6]]) om valuta te encoderen op basis van hun drie-letterige [[https://en.wikipedia.org/wiki/ISO_4217):
+Voor valuta wordt de drie-letterige [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) codering gebruikt
+binnen een [UCUM 2.1§6](https://ucum.org/ucum.html#section-Character-Set-and-Lexical-Rules) unit annotatie.
+<pre class=example highlight=json>
+    "unit": "{EUR}/h"
+</pre>
 
-`"unit": "{EUR}/h"`
+De eenheidscodering kan ook expliciet meegegeven worden door {{veld/unit}} een `object` mee te geven.
+Een {{veld/unit}} object moet de volgende attributen bevatten.
+<table dfn-type="attribute" dfn-for=unit export class="dfn-table">
+    <tr><td><dfn>type</dfn>       <td> string <td> de eenheidscodering
+    <tr><td><dfn>value</dfn>      <td> string <td> de eenheid
+</table>
 
-Het eenhedensysteem kan ook expliciet meegegeven worden door het op te slaan als `object`:
+<pre class=example highlight=json>
+    "unit": {"type": "ucum", "value": "{EUR}/h"}
+</pre>
 
-`"unit": {"type": "ucum", "value": "{EUR}/h"}`
+Zo kunnen ook andere eenheidscoderingen gebruikt worden.
+<pre class=example highlight=json>
+"unit": {"type": "pint", "value": "meter"}
+</pre>
 
-Andere system kunnen daardoor ook aangegeven worden:
+[BP]
+    Gebruik de UCUM standaard om eenheden te coderen. Niet standaard coderingssystemen worden afgeraden.
+[BPE]
 
-`"unit": {"type": "pint", "value": "meter"}`
 
-Custom unit systemen zullen echter conversie bemoeilijken en worden derhalve afgeraden door middel van een data-kwaliteits waarschuwing.
+<!--
+██    ██    ███       ███    ██     ██  ██████   ████████ ██     ██ ████ ██    ██  ██████
+███   ██   ██ ██     ██ ██   ███   ███ ██    ██  ██       ██     ██  ██  ███   ██ ██    ██
+████  ██  ██   ██   ██   ██  ████ ████ ██        ██       ██     ██  ██  ████  ██ ██
+██ ██ ██ ██     ██ ██     ██ ██ ███ ██ ██   ████ ██████   ██     ██  ██  ██ ██ ██ ██   ████
+██  ████ █████████ █████████ ██     ██ ██    ██  ██        ██   ██   ██  ██  ████ ██    ██
+██   ███ ██     ██ ██     ██ ██     ██ ██    ██  ██         ██ ██    ██  ██   ███ ██    ██
+██    ██ ██     ██ ██     ██ ██     ██  ██████   ████████    ███    ████ ██    ██  ██████
+-->
+Naamgeving {#naamgeving}
+====================================
+De data die in Amsterdam Schema wordt beschreven wordt publiek ontsloten en heeft veel externe afnemers. Het is
```

Comment:
:+1: heel nuttig dit

Reply:
Ja, zeker in zo'n lang bestand. Het is een feature van bikeshed, maar ik heb ook een extensie voor vscode gevonden, "big text".